src/mx_bluesky/I24/serial/fixed_target/i24ssx_Chip_Collect_py3v1.py

```diff
@@ -380,14 +387,24 @@ def start_i24(zebra: Zebra, parameters: FixedTargetParameters):
         )
 
         logger.debug("Arm Pilatus. Arm Zebra.")
-        yield from setup_zebra_for_fastchip_plan(
-            zebra,
-            parameters.detector_name,
-            num_gates,
-            parameters.num_exposures,
-            parameters.exposure_time_s,
-            wait=True,
-        )
+        if parameters.pump_repeat == PumpProbeSetting.Medium1:
+            yield from setup_zebra_for_fastchip_pump_probe_with_long_delays_plan(
+                zebra,
+                parameters.detector_name,
+                num_gates,
+                parameters.num_exposures,
+                parameters.exposure_time_s,
+                wait=True,
+            )
+        else:
+            yield from setup_zebra_for_fastchip_plan(
+                zebra,
+                parameters.detector_name,
+                num_gates,
+                parameters.num_exposures,
+                parameters.exposure_time_s,
+                wait=True,
+            )
```

Comment:
Could: See below for discussion on how to split up the plan, we could even then call the separate parts here e.g.
```suggestion
         start_time_offset = SHUTTER_OPENING_TIME if PumpProbeSetting.Medium1 else 0
          yield from setup_zebra_for_fastchip_plan(
              zebra,
              parameters.detector_name,
              num_gates,
              parameters.num_exposures,
              parameters.exposure_time_s,
              start_time_offset
              wait=True,
          )
        if parameters.pump_repeat == PumpProbeSetting.Medium1:
            yield from open_shutter_at_each_location(...)
```

I'm not sure if that's much cleaner or not though

Reply:
Well, having just one call to `setup_zebra_for_fastchip_plan` instead of the `if.. else..` does make it a bit more readable imo... 
I wonder if it would be a bit tidier to call `open_shutter_at_each_location` from the main `setup_zebra` plan instead of the collection one?